README.md

```diff
@@ -693,6 +693,7 @@ You can reach me on [Twitter @ileriayooo](https://twitter.com/Ileriayooo)
 | Jira          | ![Jira](https://img.shields.io/badge/jira-%230A0FFF.svg?style=for-the-badge&logo=jira&logoColor=white)                   | `![Jira](https://img.shields.io/badge/jira-%230A0FFF.svg?style=for-the-badge&logo=jira&logoColor=white)`                   |
 | Kubernetes    | ![Kubernetes](https://img.shields.io/badge/kubernetes-%23326ce5.svg?style=for-the-badge&logo=kubernetes&logoColor=white) | `![Kubernetes](https://img.shields.io/badge/kubernetes-%23326ce5.svg?style=for-the-badge&logo=kubernetes&logoColor=white)` |
 | Notion        | ![Notion](https://img.shields.io/badge/Notion-%23000000.svg?style=for-the-badge&logo=notion&logoColor=white)             | `![Notion](https://img.shields.io/badge/Notion-%23000000.svg?style=for-the-badge&logo=notion&logoColor=white)`             |
+| Packer     | ![Packer](https://img.shields.io/badge/packer-%2302A8EF.svg?style=for-the-badge&logo=packer&logoColor=white)    | `![Packer](https://img.shields.io/badge/packer-%2302A8EF.svg?style=for-the-badge&logo=packer&logoColor=white)`    |
```

Comment:
Woohooo! @thechrisandrew, thanks for making this **contribution**🚀. I'm excited to have Packer join our humble collection of markdown badges. Recently, we've been looking at the issue of accessibility with our badges #355 making sure contributions going forward adhere to certain accessibility principles like contrast. 

That said, I've got a feeling we could tweak the contrast or the outlook of this badge a little to help with the case of accessibility. Here's what I came up with.

![Packer](https://img.shields.io/badge/packer-%23FFFFFF.svg?style=for-the-badge&logo=packer&logoColor=%2302A8EF)

what do you think?

Reply:
https://coolors.co/contrast-checker/ffffff-02a8ef

the contrast score of the initial badge

Reply:
That looks awesome, quite tasteful too! One thing that kinda comes up to mind is if someone is actually using GitHub light mode, will the white background be a problem though? Other than that definitely works for me!!

Reply:
Yikes! Thanks for catching that. That would have been a total disaster. After a few dystopian hours of head-scratching, I arrived at this. Better?

![Packer](https://img.shields.io/badge/packer-%23E7EEF0.svg?style=for-the-badge&logo=packer&logoColor=%2302A8EF)

🤣 Just wondering why anyone would wanna use anything aside from dark mode

Reply:
Eyy, that will definitely work for both light and dark theme discord users! I will update the code with your revisions in a moment.

![image](https://user-images.githubusercontent.com/67407803/167252370-bda24c9b-57d1-4c05-86ea-4d9b412a5123.png)

> :rofl: Just wondering why anyone would wanna use anything aside from dark mode

I have a friend that uses light mode often, I honestly do not understand the motive behind it at all :laughing: